CMakeLists.txt

```diff
@@ -47,27 +47,15 @@ set(EXTRA_DIST_DIRS)
 include(BuildFFTW)
 include(BuildGSL)
 
-# If a CLASS installation folder is provided, use it,
-# otherwise download and compile CLASS
-if(EXTERNAL_CLASS_PATH)
-    message("Using external install of CLASS")
-    set(CLASS_LIBRARY_DIRS ${EXTERNAL_CLASS_PATH})
-    set(CLASS_INCLUDE_DIRS ${EXTERNAL_CLASS_PATH}/include)
-    set(CLASS_LIBRARIES -lclass)
-    set(CLASS_EXTERNAL True)
-else(EXTERNAL_CLASS_PATH)
-    include(BuildCLASS)
-endif(EXTERNAL_CLASS_PATH)
```

Comment:
Is it now not possible to couple to a custom version of CLASS?

Reply:
It is possible. You simply need to modify the CLASS package upstream and install it.